### PR TITLE
Create chroot.md

### PR DESCRIPTION
**Pull request by gautamkrishnar**
chroot man

### CODE REVIEW

File: pages/linux/chroot.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# chroot
+
+> Run command or interactive shell with special root directory.
+
+- General Syntax
```
Comment (rubenvereecken) [inline pages/linux/chroot.md]: Like your previous PR, we don't list general syntax.

File: pages/linux/chroot.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+- specify user and group (ID or name) to use
+
+`chroot −−userspec={{USER:GROUP}}`
+
+- specify supplementary groups as g1,g2,..,gN
+
+`chroot −−groups={{G_LIST}}`
+
+- display help and exit
```
Comment (rubenvereecken) [inline pages/linux/chroot.md]: Isn't this one a bit of a no-brainer? Usually people replace `command --help` with `tldr command`, no need to send them back. Same goes for the `--version` below.

File: pages/linux/chroot.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+- General Syntax
+
+`chroot [OPTION] NEWROOT {{COMMAND [ARG...]}}`
+
+- specify user and group (ID or name) to use
+
+`chroot −−userspec={{USER:GROUP}}`
+
+- specify supplementary groups as g1,g2,..,gN
```
Comment (rubenvereecken) [inline pages/linux/chroot.md]: I'd update this to be just "specify supplementary groups" and then handle the formatting in the line below, like `chroot --groups={{group1,group2,...}}. I do however want someone else's opinion on this one.

Comment (gautamkrishnar) [inline pages/linux/chroot.md]: chroot --groups={{group1,group2,...}
Shall i have to update this?

Comment (rubenvereecken) [inline pages/linux/chroot.md]: I had a peak at [usermod](https://github.com/tldr-pages/tldr/blob/2c215438cd3f57379fb57a8d8173897941ee8caa/pages/linux/usermod.md) where we just use `{{group1,group2}}`. I like that best, so you can safely update to that. Also keep in mind everything is lowercase, except where it's required to be caps.

**Comment by gautamkrishnar** [comment]
@rubenvereecken updated :+1:

File: pages/linux/chroot.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# chroot
+
+> Run command or interactive shell with special root directory.
+
+- Run ls command as new root directory
+
+`chroot /path/to/new/root `
```
Comment (igorshubovych) [inline pages/linux/chroot.md]:
```md
`chroot {{/path/to/new/root}}`
```

**Comment by igorshubovych** [comment]
Please:
- run `make lint`
- fix
- squash commits

**Comment by gautamkrishnar** [comment]
@igorshubovych ok now :smile: ?

File: pages/linux/chroot.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
+# chroot
+
+> Run command or interactive shell with special root directory.
+
+- Run ls command as new root directory
+
+`chroot chroot {{/path/to/new/root}} {{ls}}`
```
Comment (igorshubovych) [inline pages/linux/chroot.md]: 1. Need `chroot` once, you repeated it twice.
2. Should be either `ls` or `{{command}}`. If you choose `{{command}}`, then update the description.

**Comment by gautamkrishnar** [comment]
@rubenvereecken done :+1:

File: pages/linux/chroot.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
+# chroot
+
+> Run command or interactive shell with special root directory.
+
+- Run command as new root directory
+
+`chroot chroot {{/path/to/new/root}} {{command}}`
```
Comment (igorshubovych) [inline pages/linux/chroot.md]: You still have `chroot chroot`

**Comment by gautamkrishnar** [comment]
@rubenvereecken really sorry for that. :smile:  Now its fixed

**Comment by igorshubovych** [comment]
I am dummy with `chroot`. So excuse me if I ask something really stupid.
I just googled for `chroot` examples, use cases. I checked several pages of Google search, and I have never meet anybody using `--groups` option. So I wonder, isn't it a common option, or I googled it incorrectly.

**Comment by gautamkrishnar** [comment]
@igorshubovych i am also not sure about it. I think as it is existing in the command's man page we must include it/ 
http://www.cyberciti.biz/faq/unix-linux-chroot-command-examples-usage-syntax/

**Comment by igorshubovych** [comment]
@gautamkrishnar 
Do you use this option frequently?

**Comment by gautamkrishnar** [comment]
@igorshubovych nope...

**Comment by igorshubovych** [comment]
I would vote to remove it then.

There is no need to add just to fill the gap, if it is just 2 examples, it can still be a good page. Add only if you use this command.

**Comment by gautamkrishnar** [comment]
@rubenvereecken done :+1:

**Comment by igorshubovych** [comment]
LGTM

**Comment by felixonmars** [comment]
:+1:

**Pull request merged**